schema.sql

```diff
@@ -1,5 +1,16 @@
 /* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
     name varchar(100)
 );
```

Comment:
Kindly remove the previously added code which is not relevant to this project to keep your repo clean.